Utils.py

```diff
@@ -603,3 +603,10 @@ def is_kivy_running():
         root.withdraw()
         showerror(title, text) if error else showinfo(title, text)
         root.update()
+
+
+def title_sorted(data: typing.Sequence, key=None, ignore: typing.Set = frozenset(("a", "the"))):
+    """Sorts a sequence of text ignoring typical articles like "a" or "the"."""
+    def sorter(element: str) -> str:
+        return " ".join(filter(lambda word: word.lower() not in ignore, element.split()))
+    return sorted(data, key=lambda i: sorter(key(i)) if key else sorter(i))
```

Comment:
`key=lambda i: re.sub("^(the|a|an) ", "", i.lower()` maybe?

Reply:
I personally dislike regex as it's another language to understand to well.. understand what the code does. If there's a language-native way of doing it I'll go for it.

Reply:
Not sure that's a good argument. in web dev we have often a lot of RE, this one is rather simple, and it makes the core a lot easier to read imo.

Anyway, there is also a difference in function: the RE code only strips a single leading The, A or An, which is what we would expect: "Slay the Spire" would keep its "the", "Chapter A" would keep its "A".

That would be `element.split(" ", 1)))` in your code.

Reply:
true, it would affect things like Ori and the Blind Forest.